jni_gen/examples/README.md

```diff
@@ -0,0 +1,11 @@
+## jni_gen examples
+
+This directory contains examples on how to use jni_gen.
+
+| Directory | Description |
+| ------- | --------- |
+| [in_app_java](in_app_java/) | Demonstrates how to include custom Java code in Flutter project and call that using jni_gen |
```

Comment:
Flutter project -> Flutter Android plugin

Reply:
No it's flutter app. Not plugin. See #42 once, it details some usage modes.. For android plugin mode it's hard to get gradle dependency paths, unless you depend only on core libraries and no androidx.

Reply:
Ah you're right 👌  I was assuming the FFI plugin layout when there's an FFI plugin and then an app in the `example` subfolder.

Reply:
Let's call it a Flutter application then.